docs/words.tex

```diff
@@ -380,6 +406,8 @@ \section{Disk I/O}
 \item[saveb ( start end filenameptr filenamelength -- )] Save binary block.
 \item[device ( device\# -- )] Switches the active device.
 \item[save-forth filename ( -- )] Saves the forth to the given filename.
+\item[save-pack filename ( -- )] Saves a compact version of forth to the given filename.

```

Comment:
This could be removed from the dictionary, but is still required for the build process.  Code itself used by `save-prg`.